third_party/amd/backend/compiler.py

```diff
@@ -152,11 +156,6 @@ def get_module_map(self) -> Dict[str, ModuleType]:
     def load_dialects(self, ctx):
         amd.load_dialects(ctx)
 
-    @staticmethod
-    @functools.lru_cache()
-    def use_buffer_ops():
-        return os.environ.get("AMDGCN_USE_BUFFER_OPS", "1") == "1"
```

Comment:
Thank you! This fixes a critical path issue with an ugly solution.

Reply:
I think you missed the test that corresponds to this though. Can you make sure this test is updated by changing the value in the config instead of clearing the cache? https://github.com/triton-lang/triton/blob/e1162ee14554f693b646536b458ec3a27ed15f2f/python/test/unit/runtime/test_cache.py#L555

Reply:
Ah good catch-- let me run tests on AMD side as well, which should catch this issue

Reply:
Resolved in updated commits